### PR TITLE
Resize view on the add accounts

### PR DESCRIPTION
**Pull request by wolffan**
Adding some logic for resizing the screen as more buttons are added

### CODE REVIEW

**Comment by wolffan** [comment]
Fixes https://github.com/balancemymoney/balance-open/issues/115

**Comment by wolffan** [comment]
https://github.com/balancemymoney/balance-open/issues/115

**Comment by wolffan** [comment]
screen shot 2017-09-21 at 12 35 39

**Comment by wolffan** [comment]
@einsteinx2 this ended up having a bunch of stuff

File: Balance/Shared/Data Model/Etherscan/EthplorerAccount.swift [hunk 1]
```diff
@@ -71,7 +71,7 @@ struct EthplorerAccountObject {
     struct TokenInfo {
         let address: String
         let name: String
-        let decimals: Int
+        let decimals: Double
```
Comment (einsteinx2) [inline Balance/Shared/Data Model/Etherscan/EthplorerAccount.swift]: Why is this a double?

File: Balance/macOS/View Controllers/Add Account Workflow/OpenAddAccountViewController.swift [hunk 2]
```diff
@@ -70,6 +70,14 @@ class OpenAddAccountViewController: NSViewController {
         removeShortcutMonitor()
     }
     
+    fileprivate var screenSize: CGFloat {
```
Comment (einsteinx2) [inline Balance/macOS/View Controllers/Add Account Workflow/OpenAddAccountViewController.swift]: can you rename this to windowHeight?

File: Balance/Shared/Data Model/Etherscan/EthplorerAccount.swift (hunk 1 above)
Comment (wolffan) [reply Balance/Shared/Data Model/Etherscan/EthplorerAccount.swift]: @einsteinx2 your code just breaks if I throw an int. It's the interpretation that the checkTypes makes of the  number on a JSON file

File: Balance/macOS/View Controllers/Add Account Workflow/OpenAddAccountViewController.swift (hunk 2 above)
Comment (wolffan) [reply Balance/macOS/View Controllers/Add Account Workflow/OpenAddAccountViewController.swift]: Ok I'll do @einsteinx2

File: Balance/Shared/Data Model/Etherscan/EthplorerAccount.swift (hunk 1 above)
Comment (wolffan) [reply Balance/Shared/Data Model/Etherscan/EthplorerAccount.swift]: sorry that sounded too agressive, wasn't my intention :(

**Pull request merged**